### PR TITLE
Updates the Uyuni install server page

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

This updates the Uyuni install server page to remove the 15.1 reference, and to more clearly explain the step where the variables are set (per https://github.com/uyuni-project/uyuni-docs/pull/430). Also removes some extraneous newlines.

## Still to do
- [ ] Investigate the Uyuni "Quickstart" documentation (https://www.uyuni-project.org/pages/stable-version.html) @jcayouette 

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [ ] Manager-4.2
- [x] Uyuni

# Links

Partial: https://github.com/uyuni-project/uyuni-docs/issues/428

### CODE REVIEW

File: modules/installation/pages/install-uyuni.adoc [hunk 1]
```diff
@@ -7,39 +7,33 @@ For requirements, see xref:installation:uyuni-install-requirements.adoc[].
 
 
 
-== Install Uyuni on openSUSE Leap 15.1
+== Install Uyuni on openSUSE Leap 15
 
 .Procedure: Installing openSUSE Leap 15.1 with Uyuni
```
Comment (keichwa) [inline modules/installation/pages/install-uyuni.adoc:12]:
```suggestion
.Procedure: Installing openSUSE Leap 15 with Uyuni
```

**Comment by Loquacity** [comment]
> I'd even remove the general "15" version info. Instead we should mention somewhere that you must the latest Leap version and that you should check the release notes for more detailed version info.

That's a good idea. Lemme update the PR.

File: modules/installation/pages/install-uyuni.adoc [hunk 2]
```diff
@@ -5,13 +5,15 @@
 
 For requirements, see xref:installation:uyuni-install-requirements.adoc[].
 
+For more information about the latest version and updates of openSUSE Leap, see https://doc.opensuse.org/release-notes/.
```
Comment (keichwa) [inline modules/installation/pages/install-uyuni.adoc:8]: I think it even deserve an admon :)

```suggestion
[NOTE]
====
For more information about the latest version and updates of openSUSE Leap, see https://doc.opensuse.org/release-notes/.
====
```

Comment (Loquacity) [reply modules/installation/pages/install-uyuni.adoc:8]: Sure, why not? ;)

**Pull request merged**